smart_campus/app/views.py

```diff
@@ -384,11 +418,11 @@ def get_unanswered_question(request):
     questions = Question.objects.filter(linked_station=station)
     user_answered_questions = Question.objects.filter(user=user)
 
-    # Get questions that the user hasn't answered yet
+    """Get questions that the user hasn't answered yet"""
     not_answered_questions = questions.difference(user_answered_questions)
 
     if not_answered_questions.exists():
-        # pick 1 question randomly
+        """pick 1 question randomly"""
```

Comment:
It seems you misunderstand.
You **should not** use `"""` for comments.
Docstrings are those written at the beginning of a code block that describes the code block. 

e.g.
```python
def some_func():
    """This is the docstring for some_func"""
    a = 1
    b = 1
    # This is a comment
    return a + b
```

You should update it elsewhere as well.

Reply:
Learned it.